.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ jobs:
   build:
     <<: *defaults
     docker:
-      - image: circleci/node:14.15.0
+      - image: circleci/node:16.7-browsers
```

Comment:
Maybe we should bump in package too?

Reply:
good point.

Reply:
Thanks. Updated.

Reply:
nice, make sure to do some sanity testing, could have some side-effects